app/components/collections/show/details_header_component.html.erb

```diff
@@ -4,7 +4,7 @@
   <% end %>
   <div class="row">
     <div class="col"> </div>
-    <div class="col-6" data-controller="work-type purl-reservation">
+    <div class="col-6" data-controller="work-type work-type-modal purl-reservation">
```

Comment:
The work type functionality was split into 3 different controllers for base work type modal functionality, select work type when creating work functionality, and change work type admin functionality.

This required adding in additional controllers in various places for this change.